source/_integrations/backup.markdown

```diff
@@ -31,3 +31,38 @@ You need to restart Home Assistant after you add this configuration.
 When it has started up again you will find a new "Backup" entry in the main menu (**Configuration** -> **Backup**).
 
 The backup files are stored in a new "backups" subdirectory in the root of your configuration directory.
+
+## Services
+
+The update entity exposes a services that can be used to automate the backup
```

Comment:
- This is the backup entity, not the update entity
- "Services" is plural, when it should be singular

```
The backup entity exposes a service that can be used to automate the backup process.
```

Reply:
Copy/Pasta! 🎉 

```suggestion
The backup integration exposes a service that can be used to automate the backup
```

Thanks, @hkamran80 👍